frontend/src/components/MapView/Layers/StaticRasterLayer/index.test.tsx

```diff
@@ -0,0 +1,42 @@
+import timezoneMock from 'timezone-mock';
+import { createStaticRasterLayerUrl } from '.';
+
+const f = () => {
+  const result = createStaticRasterLayerUrl(
+    createUrlTestData.baseUrl,
+    createUrlTestData.dates,
+    createUrlTestData.selectedDate,
+  );
+
+  expect(result).toBe(createUrlTestData.result);
+};
+
+describe('createStaticRasterLayerUrl', () => {
+  afterAll(() => {
+    timezoneMock.unregister();
+  });
+
+  test('Should work with UTC', () => {
+    timezoneMock.register('UTC');
+    f();
+  });
+
+  test('Should work with US/Pacific', () => {
+    timezoneMock.register('US/Pacific');
```

Comment:
Maybe add a test with a timezone on the east side of UTC? like UTC+6 or whatever. Ideally, we'd have a list of all timezones, and run the test in a loop for each one. Maybe something like https://jestjs.io/docs/api#testeachtablename-fn-timeout ?

Reply:
Let's do india standard time, it's both ahead of UTC and 30min off @echaidemenos